tools/meteor-services/auth.js

```diff
@@ -65,6 +65,9 @@ var loggedInAccountsConnection = function (token) {
     connection.close();
 
     if (err.error === 403) {
+      // Better than saying nothing at all
+      Console.error("warning: login token was rejected by the server");
```

Comment:
Right now, adding this message here would actually double up error messages in some places, [like here](https://github.com/meteor/meteor/blob/fe143dd84752bc3dea42ea35d44c2d4e93ba14f5/tools/cli/commands.js#L1904-L1907).

I think there might be a better place to put this, [maybe here?](https://github.com/meteor/meteor/blob/fe143dd84752bc3dea42ea35d44c2d4e93ba14f5/tools/meteor-services/auth.js#L620-L676)

What was the actual solution to your issue? Perhaps that could be explained too.